lib/escape.c

```diff
@@ -76,8 +86,10 @@ char *curl_unescape(const char *string, int length)
   return curl_easy_unescape(NULL, string, length, NULL);
 }
 
-char *curl_easy_escape(struct Curl_easy *data, const char *string,
-                       int inlength)
+/* allow for custom definition of unreserved characters */
+char *curl_easy_escape_flexible(struct Curl_easy *data, const char *string,
```

Comment:
this is a private function, right? It should be static and not use the `curl_` prefix.